openshift/topics/get_started.adoc

```diff
@@ -132,10 +150,15 @@ $ oc new-app --template={project_templates_version}-x509-https
     Run 'oc status' to view your app.
 ----
 
-Alternatively, perform one of the following procedures to deploy the {project_name} template by using the OpenShift 3.x or 4.x web console.
-
+[id="deploy3x"]
 ===== Deploying the Template using the OpenShift 3.x Web Console
 
+.Prerequisites
+
+* Perform the steps described in xref:image-streams-applications-templates[Using the {project_openshift_product_name} Image Streams and Application Templates].
+
+.Procedure
+
 . Log in to the OpenShift web console and select the _sso-app-demo_ project space.
```

Comment:
Should this be changed to `*sso-app-demo*` too? (since looks we are using bold highlighted text below).